helm_deploy/values-dev.yaml

```diff
@@ -16,35 +15,9 @@ generic-service:
     AUTHORISED_ROLES: ROLE_PRISONS_REPORTING_USER
 
   allowlist:
-    office: "217.33.148.210/32"
-    health-kick: "35.177.252.195/32"
-    petty-france-wifi: "213.121.161.112/28"
-    global-protect: "35.176.93.186/32"
-    mojvpn: "81.134.202.29/32"
-    cloudplatform-live-1: "35.178.209.113/32"
-    cloudplatform-live-2: "3.8.51.207/32"
-    cloudplatform-live-3: "35.177.252.54/32"
-    circleci-1: "3.228.39.90"
-    circleci-2: "18.213.67.41"
-    circleci-3: "34.194.94.201"
-    circleci-4: "34.194.144.202"
-    circleci-5: "34.197.6.234"
-    circleci-6: "35.169.17.173"
-    circleci-7: "35.174.253.146"
-    circleci-8: "52.3.128.216"
-    circleci-9: "52.4.195.249"
-    circleci-10: "52.5.58.121"
-    circleci-11: "52.21.153.129"
-    circleci-12: "52.72.72.233"
-    circleci-13: "54.92.235.88"
-    circleci-14: "54.161.182.76"
-    circleci-15: "54.164.161.41"
-    circleci-16: "54.166.105.113"
-    circleci-17: "54.167.72.230"
-    circleci-18: "54.172.26.132"
-    circleci-19: "54.205.138.102"
-    circleci-20: "54.208.72.234"
-    circleci-21: "54.209.115.53"
+    groups:
```

Comment:
Previously a big chunk of prisons would have been added to this, due to inheriting via yaml merging... to preserve that behaviour you should add the prisons group here. 
 
If you want individual allowlists then suggest removing from values.yaml and just managing in each different env file as this is a bit misleading

Alternatively, if you want to use the same allowlist in each env then just keep the values.yaml version. 
if you want a chat about this then please get in touch!

Reply:
Makes sense to me.